model/src/main/kotlin/config/FileListStorageConfiguration.kt

```diff
@@ -29,33 +29,33 @@ import org.ossreviewtoolkit.utils.ort.ortDataDirectory
 import org.ossreviewtoolkit.utils.ort.storage.FileStorage
 import org.ossreviewtoolkit.utils.ort.storage.LocalFileStorage
 
-private const val TABLE_NAME = "provenance_file_listings"
-private const val FILENAME = "file_listings.xz"
+private const val TABLE_NAME = "provenance_file_lists"
+private const val FILENAME = "file_lists.xz"
```

Comment:
This should also be changed to singular because it contains only one file list, right?